pkg/operator/certrotation/signer_test.go

```diff
@@ -50,13 +51,24 @@ func TestEnsureSigningCertKeyPair(t *testing.T) {
 				if len(actual.Data["tls.crt"]) == 0 || len(actual.Data["tls.key"]) == 0 {
 					t.Error(actual.Data)
 				}
+				if len(actual.Annotations) == 0 {
+					t.Errorf("expected certificates to be annotated")
+				}
```

Comment:
This looks redundant, any value for annotations that would fail here is also going to fail below.

Reply:
Yes, but I'd prefer to a clear error message

Reply:
Fine with me given that nothing out of the control of this test should be touching annotations.